src/cli/commands/create.js

```diff
@@ -16,17 +16,50 @@ export function hasWrapper(commander: Object, args: Array<string>): boolean {
   return true;
 }
 
+const coerceName = (name = '') => (name === '' ? 'create' : `create-${name}`);
+const coerceScope = (scope = '') => (scope === '@' ? '' : scope);
+const coerceFullName = (scope, name) => [coerceScope(scope), coerceName(name)].filter(Boolean).join('/');
+
+/**
+ * # Tests
+ *
+ * ## basic
+ * parseBuilderName('name').packageName === 'create-name'
+ * parseBuilderName('@scope/name').packageName === '@scope/create-name'
+ *
+ * ## not adding "-" if name is empty
+ * parseBuilderName('@scope/').packageName === '@scope/create'
+ * parseBuilderName('@scope').packageName === '@scope/create'
+ *
+ * ## edge cases
+ * parseBuilderName('@/name').packageName === 'create-name'
+ * parseBuilderName('/name').packageName === 'create-name'
+ * parseBuilderName('@/').packageName === 'create'
+ */
+const parseBuilderName = str => {
+  const parts = str.split('/');
+  if (parts.length === 1 && !str.includes('@')) {
```

Comment:
`str.charAt(0) === '@'` would be more correct, I think.

Reply:
right

Reply:
fixed